magprime/templates/regextra.html

```diff
@@ -0,0 +1,29 @@
+<script type="text/javascript">
+    $(function () {
```

Comment:
Good to know.  Based on that feedback, in cases where I'm not sure whether I should write a custom utility method or just use basic jQuery I'll err on the side of doing the latter.